conjure-java-core/src/test/java/com/palantir/conjure/java/TestBase.java

```diff
@@ -56,7 +56,7 @@ protected static void validateGeneratorOutput(List<Path> files, Path outputDir,
                 Files.deleteIfExists(output);
                 Files.copy(file, output);
             }
-            assertThat(readFromFile(file)).isEqualTo(readFromFile(output));
+            assertThat(readFromFile(file)).describedAs(output.toString()).isEqualTo(readFromFile(output));
```

Comment:
Unsure if you're aware, we have a helper function to update the expected output `./gradlew test -Drecreate=true` documented [here](https://github.com/palantir/conjure-java/blob/develop/CONTRIBUTING.md#local-development)